cli/jira_config_gen/__init__.py

```diff
@@ -39,16 +39,33 @@
     default="/tmp/jira.config",
     type=click.STRING,
 )
+@click.option(
+    "--template_dir",
+    help="Directory holding templates",
+    required=True,
```

Comment:
Since you have `default`, `required` is not needed here.

Reply:
I will fix this on the other commands as well